views/administration/ldap-edit.hbs

```diff
@@ -35,6 +35,9 @@ Probably delete it.--}}
                 Inhalte werden nicht übernommen, da bei dem Nutzer der Vorname, der Nachname und/oder die E-Mail-Adresse
                 nicht im LDAP verfügbar ist.
             </p>
+            <p>
+                Weitere Informationen zur LDAP-Konfiguration finden Sie in unserem <a href="https://docs.schul-cloud.org/pages/viewpage.action?pageId=55902270">Hilfebereich</a>.
```

Comment:
wir könnten auch auf unseren eigenen Hilfebereich verlinken
`https://schul-cloud.org/help/confluence/55902270` bzw. `/help/confluence/55902270`
Dann bleibt man in der Schul-Cloud

Reply:
Ah cool, das wusste ich nicht!